src/external/lua/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
```

Comment:
On  8/01/16 10:31 AM, Pedro Côrte-Real wrote:
> In src/external/lua/CMakeLists.txt <https://github.com/darktable-org/darktable/pull/1039#discussion_r49145862>:
>
>> @@ -1,4 +1,4 @@
>> -cmake_minimum_required(VERSION 2.6)
>
> The current Ubuntu LTS (14.04) still has 2.8.12 so it would be nice if we could keep compatibility for that until the next LTS (16.04) is out.


Solaris 11.3 still has 2.8.6. Yes, not a primary development
target, but I do have users (apart from me) of this build.
I'd prefer to not have to maintain my own copy of cmake as
well as all the gnome bits :)


thanks,
James C. McPherson
--
Solaris kernel software engineer, system admin and troubleshooter
               https://www.jmcpdotcom.com/blog
Find me on LinkedIn @ http://www.linkedin.com/in/jamescmcpherson